modules/shape/test/test_shape.cpp

```diff
@@ -104,8 +107,8 @@ class ShapeBaseTest : public cvtest::BaseTest
 
         // Uniformly sampling
         random_shuffle(contoursQuery.begin(), contoursQuery.end());
-        int nStart=NP;
         vector<PointType> cont;
+        int nStart=NP;
```

Comment:
it is not indeed ... I was trying an alternative solution and forgot to remake it original state.